src/dev/java/me/moodcat/database/BulkInsertApp.java

```diff
@@ -39,10 +44,10 @@ public static void main(final String[] args) throws Exception {
      *             when the bulkinsertion has failed.
      */
     public void run() throws Exception {
-        final Injector injector = Guice.createInjector(new BulkInsertServletModule());
-        final PersistService persistService = injector.getInstance(PersistService.class);
+        Injector injector = Guice.createInjector(new BulkInsertServletModule());
+        PersistService persistService = injector.getInstance(PersistService.class);
         persistService.start();
-        final BulkInsertData bulkInsertData = injector.getInstance(BulkInsertData.class);
```

Comment:
As this is only queried on initialisation, the messages are needed anyway. But as many to one relations have to be queried anyway, I doubt if this is any faster.